measurement.js

```diff
@@ -21,12 +22,15 @@
   const pbjs = (window.pbjs = window.pbjs || { que: [] });
   const googletag = (window.googletag = window.googletag || { cmd: [] });
 
-  const TREATMENT_RATE = 0.95;
-  if (window.liModuleEnabled !== undefined) {
+  // Set or upgrade the treatment rate and ModuleEnabled flags
+  if ((window.liTreatmentRate || 0) < LI_TREATMENT_RATE && !window.liModuleEnabled) {
+    // If a previous wrapper set a lower treatment rate, adjust the probability to upgrade
+    // and re-flip if needed. Otherwise, match LI_TREATMENT_RATE.
+    const treatment_rate = 1 - (1 - LI_TREATMENT_RATE) / (1 - (window.liTreatmentRate || 0));
```

Comment:
Please correct me if i am wrong. I think we decided to declare one holdout by the first script and keep it uniform for all the scripts and discussed that we could make the holdout 98-2 or 97-3 instead of 95-5 if we feel 95-5 is too high. That way the holdout percentage is uniform and transparent to everyone and no one is surprised at the time of billing.